### ISSUE TITLE
The plugin seems to add an extra `n` after the bing int literal value

### ISSUE THREAD

**Issue by jraoult** [GoogleChromeLabs/babel-plugin-transform-jsbi-to-bigint#10]
I wonder if I'm missing something obvious, but even a simple example using the babel plugin generates a wrong output:

```js
import JSBI from "jsbi";

const BIG_INT_0 = JSBI.BigInt(0);
```

generates:

```js
const BIG_INT_0 = 0nn;
```

See this [babel playground](https://babeljs.io/repl#?browsers=last%201%20chrome%20version&build=&builtIns=false&corejs=false&spec=false&loose=false&code_lz=JYWwDg9gTgLgBAKQMoCECScBmUIjgIgCsBnAI2HwG4AoagYwgDtj50BxAfTQDkAVDgAxwAvIlRoAdCmABzNIxgAKAQEpKQA&debug=false&forceAllTransforms=false&shippedProposals=false&circleciRepo=&evaluate=true&fileSize=false&timeTravel=false&sourceType=module&lineWrap=true&presets=&prettier=false&targets=&version=7.14.4&externalPlugins=babel-plugin-transform-jsbi-to-bigint%401.3.1).

I'm not a babel expert but I wouldn't be surprised if the `n` was redundant here: https://github.com/GoogleChromeLabs/babel-plugin-transform-jsbi-to-bigint/blob/1187d6fad826dc9369f40c77f5c640f6f53432eb/src/index.js#L104

**Comment by mathiasbynens** [comment]
I suppose this changed upstream in Babel — the code certainly used to work before. Open to a patch that updates to the latest versions :)

**Comment by jraoult** [comment]
@mathiasbynens I just opened a PR. Let me know if you need anything else from me

**Comment by hugbubby** [comment]
@mathiasbynens 
The issue still occurs for me on 1.4.0. Doesn't seem to have been fixed.

**Comment by mathiasbynens** [comment]
Fixed in v1.4.0 thanks to @jraoult!
